src/referencePackages/src/dir.props

```diff
@@ -1,30 +1 @@
-<?xml version="1.0" encoding="utf-8"?>
-<Project>
-  <Import Project="$(RepoRoot)artifacts/source-built/PackageVersions.props" />
-  <PropertyGroup>
-    <!-- Common properties for all Reference Packages -->
-    <DebugType>None</DebugType>
-    <DebugSymbols>false</DebugSymbols>
-    <DisableImplicitFrameworkReferences>true</DisableImplicitFrameworkReferences>
-    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
-
-    <AssemblyOriginatorKeyFile>$(RepoRoot)src/referencePackages/MSFT.snk</AssemblyOriginatorKeyFile>
-    <KeyFileDir>$(RepoRoot)src/referencePackages/snk/</KeyFileDir>
-    <SignAssembly>true</SignAssembly>
-    <PublicSign>true</PublicSign>
-
-    <SourceControlInformationFeatureSupported>false</SourceControlInformationFeatureSupported>
-    <Configuration>Release</Configuration>
-    <EnableSourceLink>false</EnableSourceLink>
-
-    <NoStdLib>true</NoStdLib>
-    <NoConfig>true</NoConfig>
-    <AddAdditionalExplicitAssemblyReferences>false</AddAdditionalExplicitAssemblyReferences>
-    <GenerateTargetFrameworkAttribute>false</GenerateTargetFrameworkAttribute>
-    <RuntimeMetadataVersion>v4.0.30319</RuntimeMetadataVersion>
-
-    <EnableDefaultCompileItems>false</EnableDefaultCompileItems>
-    <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
-    <WarningsAsErrors></WarningsAsErrors>
-  </PropertyGroup>
-</Project>
+<Project />
```

Comment:
Is this file still needed?

Reply:
It's referenced in every csproj, keeping this file to avoid a mass edit for now. I'll add a comment.